test/uk/gov/hmrc/pillar2stubs/controllers/FinancialDataControllerSpec.scala

```diff
@@ -57,7 +56,7 @@ class FinancialDataControllerSpec extends AnyFreeSpec with Matchers with GuiceOn
       val request:         FakeRequest[AnyContentAsEmpty.type] = buildFakeRequest(invalidIdNumber)
       val result:          Future[Result]                      = route(app, request).value
 
-      status(result) mustBe BAD_REQUEST
+      status(result) mustBe play.api.http.Status.BAD_REQUEST
```

Comment:
Could you please switch this to an import as requested in https://github.com/hmrc/pillar2-stubs/pull/100#discussion_r2477202376? If you disagree then that's fine, but it'd be good to get this resolved.